libgcc/config.host

```diff
@@ -608,6 +608,9 @@ x86_64-*-freebsd*)
 	tmake_file="${tmake_file} i386/t-freebsd i386/t-crtstuff"
 	md_unwind_header=i386/freebsd-unwind.h
 	;;
+i[34567]86-*-hipperos*)
+	tmake_file="$tmake_file i386/t-crtstuff t-crtstuff-pic t-libgcc-pic"
```

Comment:
Why libgcc?

Reply:
GCC simply does not compile if you do not configure libgcc for the target.